sdk/test/UnitTests/TestLambdaContext.cs

```diff
@@ -22,12 +22,19 @@
 using Amazon.XRay.Recorder.Core.Sampling;
 using Amazon.XRay.Recorder.Core.Internal.Context;
 using Amazon.XRay.Recorder.Core.Internal.Utils;
+using System.Net;
+using System.Security.Policy;
+using Amazon.XRay.Recorder.Handlers.System.Net;
+using Moq;
+using Amazon.XRay.Recorder.Core.Internal.Emitters;
+using Amazon.XRay.Recorder.UnitTests.Tools;
 
 namespace Amazon.XRay.Recorder.UnitTests
 {
     [TestClass]
     public class TestLambdaContext : TestBase
     {
+        private const string URL = "https://httpbin.org/";
```

Comment:
Rename to Mock URL to clarify that this URL is used to mock an HTTP request. 
Is there a specific reason this was set as a class variable and not a local variable in the test it is being used in?

Reply:
Done